packages/@react-types/shared/src/dom.d.ts

```diff
@@ -58,7 +58,9 @@ export interface FocusableDOMProps extends DOMProps {
    * be avoided except in rare scenarios where an alternative means of accessing
    * the element or its functionality via the keyboard is available.
    */
-  excludeFromTabOrder?: boolean
+  excludeFromTabOrder?: boolean,
+  /** A tabIndex to return. */
+  tabIndex?: number
```

Comment:
This shows that we'll have 2 competing props - `excludeFromTabOrder` and `tabIndex` which both "control" the returned `tabIndex`. I would say this is not ideal - maybe `excludeFromTabOrder` should be deprecated in the future?